### PR TITLE
solution

### PR DESCRIPTION
**Pull request by kovaliskoveronika**

### CODE REVIEW

**Review by RedCatHot — APPROVED** [review]
It is possible that not all files are needed in the PR, for example, such as library_managment.db

File: main.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+@app.post("/authors/", response_model=schemas.AuthorCreate)
+def create_author(
+    author: schemas.AuthorCreate,
+    db: Session = Depends(get_db),
+):
+    db_author = crud.create_author(db=db, author=author)
+    return db_author
+
+
+@app.get("/authors/{author_id}", response_model=schemas.Author)
```
Comment (Oleg-s-s) [inline main.py]: /authors/{author_id}/

File: main.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        )
+    return db_author
+
+
+@app.get("/books/", response_model=list[schemas.Book])
+def read_books(db: Session = Depends(get_db), skip: int = 0, limit: int = 100):
+    return crud.get_books(db=db, skip=skip, limit=limit)
+
+
+@app.get("/authors/{author_id}/book", response_model=list[schemas.Book])
```
Comment (Oleg-s-s) [inline main.py]: /

File: main.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        raise HTTPException(
+            status_code=400,
+            detail="There is no such author"
+        )
+
+    db_book = crud.get_books_by_authors_id(db, author_id, skip, limit)
+    return db_book
+
+
+@app.post("/authors/{author_id}/book", response_model=schemas.BookCreate)
```
Comment (Oleg-s-s) [inline main.py]: /

**Review by Oleg-s-s — CHANGES_REQUESTED** [review]

File: library_management.db
Comment (Oleg-s-s) [inline library_management.db:1]: rm from pr

**Review by LiudmylaKulinchenko — APPROVED** [review]
Well done

**Pull request closed without merging**